src/applications/personalization/profile360/components/PaymentInformationEditModal.jsx

```diff
@@ -118,9 +118,11 @@ class PaymentInformationEditModal extends React.Component {
         onClose={this.props.onClose}
       >
         {!!this.props.responseError && (
-          <PaymentInformationEditModalError
-            responseError={this.props.responseError}
-          />
+          <div id="errors" role="alert" aria-atomic="true">
```

Comment:
Trevor said that this `div` should always be on the page and the Alert should be conditionally added within it. So this `dvi` should wrap the conditional logic started on line 120

Reply:
Done